ci/e2e-tests.sh

```diff
@@ -654,7 +654,7 @@ ssh -i "$PWD/vm-ssh-key" "aziot@$vm_public_ip" "
     printf 'Device twin: %s\n' \"\$device_twin\" >&2
 
     module_identity=\"\$(
-        curl --unix-socket '/run/aziot/identityd.sock' \\
+        sudo curl --unix-socket '/run/aziot/identityd.sock' \\
```

Comment:
This was only done because `/identities/identity` returned the device identity. Since you've fixed it now, change this to invoke that API and you don't need `sudo`

Reply:
Good call. Done.